spotinst/commons/administration_org_programmatic_user/fields_spotinst_administration_org_programmatic_user.go

```diff
@@ -0,0 +1,312 @@
+package administration_org_programmatic_user
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spotinst/spotinst-sdk-go/service/administration"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/terraform-provider-spotinst/spotinst/commons"
+)
+
+func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
+
+	fieldsMap[Name] = commons.NewGenericField(
+		commons.AdministrationOrgProgrammaticUser,
+		Name,
+		&schema.Schema{
+			Type:     schema.TypeString,
+			Required: true,
```

Comment:
re-check if it's required or optional param

Reply:
Required tested thru postman